spec/support/factory_helper.rb

```diff
@@ -3,7 +3,8 @@
 module AnsibleTowerClient
   module FactoryHelper
     def self.stringify_attribute_keys(hash)
-      [:description, :extra_vars, :id, :instance_id, :inventory, :name, :results, :type, :url].each do |attribute|
+      [:description, :extra_vars, :id, :instance_id, :inventory, :name, :results,
```

Comment:
If the array needs to be wrapped, it should probably be in a constant at the top.